src/axom/slic/docs/sphinx/sections/synopsis.rst

```diff
@@ -0,0 +1,59 @@
+.. ## Copyright (c) 2017-2019, Lawrence Livermore National Security, LLC and
+.. ## other Axom Project Developers. See the top-level COPYRIGHT file for details.
+.. ##
+.. ## SPDX-License-Identifier: (BSD-3-Clause)
+
+Slic is a component of the `Axom Toolkit`_, currently in development at
+`Lawrence Livermore National Laboratory (LLNL) <http://www.llnl.gov>`_.
+Slic provides a *light-weight*, *modular* and *extensible* logging
+infrastructure that simplifies logging application messages.
+
+**Key Features**
+
+Some of the features of Slic include the following:
```

Comment:
The wording of several items in the list below describe design points more so than features. See comments below.